projects/packages/forms/src/contact-form/js/accessible-form.js

```diff
@@ -354,9 +373,17 @@ const createSpinner = () => {
  */
 const createWarningIcon = () => {
 	const elt = document.createElement( 'span' );
+	const srOnly = document.createElement( 'span' );
+	const icon = document.createElement( 'i' );
+
+	srOnly.textContent = L10N.warning;
```

Comment:
I've updated the function implementation to make sure the exclamation mark is not read by screen readers.